### PR TITLE
[full-ci] [tests-only] Update expected failures after core #38876

### PR DESCRIPTION
**Pull request by kiranparajuli589**
### Description
This PR update scenarios in the expected failures file with correct issue tags.
- closed issues have been replaced with new issues
- closed issues: #1303 & https://github.com/owncloud/product/issues/207

### CODE REVIEW

**Comment by phil-davis** [comment]
@kiranparajuli589 https://github.com/owncloud/core/pull/38876 has been merged. You can make an ordinary PR to update the core commit id.

File: config/identifier-registration.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
```
Comment (phil-davis) [inline config/identifier-registration.yaml:1]: @kiranparajuli589 is this an accident?

Comment (kiranparajuli589) [reply config/identifier-registration.yaml]: yes it was, reverted!

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/5650/37/7
```
runsh: There were no unexpected failures.
runsh: Total unexpected passed scenarios throughout the test run:
apiTrashbin/trashbinRestore.feature:110
apiTrashbin/trashbinRestore.feature:111
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/5654/37/7
```
runsh: Total unexpected failed scenarios throughout the test run:
apiTrashbin/trashbinRestore.feature:110
apiTrashbin/trashbinRestore.feature:111
runsh: Total unexpected passed scenarios throughout the test run:
apiTrashbin/trashbinRestore.feature:108
```

  @skipOnOcV10 @issue-35974
  Scenario Outline: restoring a file to an already existing path overrides the file                              # /srv/app/testrunner/tests/acceptance/features/apiTrashbin/trashbinRestore.feature:96
    Given user "Alice" has uploaded file with content "file to delete" to "/.hiddenfile0.txt"                    # FeatureContext::userHasUploadedAFileWithContentTo()
    And using <dav-path> DAV path                                                                                # FeatureContext::usingOldOrNewDavPath()
    And user "Alice" has created folder "/PARENT"                                                                # FeatureContext::userHasCreatedFolder()
    And user "Alice" has uploaded file with content "PARENT file content" to <upload-path>                       # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Alice" has deleted file <delete-path>                                                              # FeatureContext::userHasDeletedFile()
    When user "Alice" restores the file with original path <delete-path> to <upload-path> using the trashbin API # TrashbinContext::userRestoresTheFileWithOriginalPathToUsingTheTrashbinApi()
    Then the HTTP status code should be "204"                                                                    # FeatureContext::thenTheHTTPStatusCodeShouldBe()
    And as "Alice" file <upload-path> should exist                                                               # FeatureContext::asFileOrFolderShouldExist()
    And the content of file <upload-path> for user "Alice" should be "file to delete"                            # FeatureContext::contentOfFileForUserShouldBe()

    Examples:
      | dav-path | upload-path                | delete-path        |
      | old      | "/PARENT/textfile0.txt"    | "/textfile0.txt"   |
      | new      | "/PARENT/textfile0.txt"    | "/textfile0.txt"   |
        The downloaded content was expected to be 'file to delete', but actually is 'PARENT file content'. HTTP status was 200
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'file to delete'
        +'PARENT file content'
      | old      | "/PARENT/.hiddenfile0.txt" | ".hiddenfile0.txt" |
        The downloaded content was expected to be 'file to delete', but actually is 'PARENT file content'. HTTP status was 200
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'file to delete'
        +'PARENT file content'
      | new      | "/PARENT/.hiddenfile0.txt" | ".hiddenfile0.txt" |
        The downloaded content was expected to be 'file to delete', but actually is 'PARENT file content'. HTTP status was 200
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'file to delete'
        +'PARENT file content'
```

That scenario is intermittent. Sometimes the file is correctly restored, overwriting the existing file in `/PARENT`. But sometimes the content is not updated. But always the correct 204 response is given.

This is the same behaviour as in core issue https://github.com/owncloud/core/issues/35974

That is very strange that this intermittent behaviour happens in oC10 core (implemented with PHP and database), and also in OCIS with "owncloud" storage (i,plementated with Go and file-system-based storage of attributes etc)

**Comment by phil-davis** [comment]
In `cs3org/reva` PR https://github.com/cs3org/reva/pull/1795 these scenarios failed consistently on "owncloud" storage.
https://drone.cernbox.cern.ch/cs3org/reva/1881/17/7

What is going on?

@C0rby you might be familiar with this. Any ideas?

**Comment by C0rby** [comment]
Ah, damn... this is bad...
I don't know why this is happening but I'll take a look into the owncloud storage driver.

**Comment by C0rby** [comment]
@phil-davis, I think I found the issue and opened a PR here: https://github.com/cs3org/reva/pull/1833.

**Comment by phil-davis** [comment]
https://github.com/cs3org/reva/pull/1833 looks good - fixes the problem. Let's see how quickly that can be merged and then find its way into `owncloud/ocis` - then we will not need to mess about with how to skip the scenario.

**Comment by phil-davis** [comment]
I adjusted the core commit id to the latest, and put back the `trashbinRestore` expected-failures. We do sometimes (most of the time) have those `trashbinRestore` scenarios fail. So we might get a "pass" in CI. (Other PRs are often getting a pass!)

It would be good to get this merged. It does not make any difference to the issue of https://github.com/cs3org/reva/pull/1833 - that can be fixed in OCIS independently.

**Comment by phil-davis** [comment]
There were a lot of fails in the last CI run, all with `owncloud` storage:
https://drone.owncloud.com/owncloud/ocis/5663/25/7
Core-API-Tests-owncloud-storage-2
runsh: Total unexpected failed scenarios throughout the test run:
apiMain/checksums.feature:119
apiMain/checksums.feature:132

https://drone.owncloud.com/owncloud/ocis/5663/31/7
Core-API-Tests-owncloud-storage-5
runsh: Total unexpected failed scenarios throughout the test run:
apiShareOperationsToShares1/accessToShare.feature:24
apiShareOperationsToShares1/accessToShare.feature:25
apiShareOperationsToShares1/accessToShare.feature:41
apiShareOperationsToShares1/accessToShare.feature:42
apiShareOperationsToShares1/accessToShare.feature:58
apiShareOperationsToShares1/accessToShare.feature:59
apiShareOperationsToShares1/accessToShare.feature:74
apiShareOperationsToShares1/accessToShare.feature:75
apiShareOperationsToShares1/gettingShares.feature:39
apiShareOperationsToShares1/gettingShares.feature:40
apiShareOperationsToShares1/gettingShares.feature:62
apiShareOperationsToShares1/gettingShares.feature:63
apiShareOperationsToShares1/gettingShares.feature:85
apiShareOperationsToShares1/gettingShares.feature:86
apiShareOperationsToShares1/gettingShares.feature:106
apiShareOperationsToShares1/gettingShares.feature:107
apiShareOperationsToShares1/gettingShares.feature:188

https://drone.owncloud.com/owncloud/ocis/5663/33/7
Core-API-Tests-owncloud-storage-6
runsh: Total unexpected failed scenarios throughout the test run:
apiShareReshareToShares1/reShare.feature:61
apiShareReshareToShares1/reShare.feature:62
apiShareReshareToShares1/reShare.feature:78
apiShareReshareToShares1/reShare.feature:79
apiShareReshareToShares1/reShare.feature:95
apiShareReshareToShares1/reShare.feature:96
apiShareReshareToShares1/reShare.feature:112
apiShareReshareToShares1/reShare.feature:113

https://drone.owncloud.com/owncloud/ocis/5663/35/7
Core-API-Tests-owncloud-storage-7
runsh: Total unexpected failed scenarios throughout the test run:
apiShareUpdateToShares/updateShare.feature:29
apiShareUpdateToShares/updateShare.feature:30
apiShareUpdateToShares/updateShare.feature:77
apiShareUpdateToShares/updateShare.feature:78
apiShareUpdateToShares/updateShare.feature:256
apiShareUpdateToShares/updateShare.feature:257
apiShareUpdateToShares/updateShare.feature:394
apiShareUpdateToShares/updateShare.feature:395
apiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature:34
apiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature:35
apiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature:54

I restarted - I am not sure what happened!

**Review by phil-davis — APPROVED** [review]

**Pull request merged**